ui/src/common/ArtistShow.js

```diff
@@ -0,0 +1,388 @@
+            </Typography>
+          </div>
+          <Card className={classes.artDetail}>
+            <div className={classes.details}>
+              <CardContent className={classes.content}>
+                <Typography component="h5" variant="h5">
+                  {title}
+                </Typography>
+                <Collapse
+                  collapsedHeight={'1.5em'}
+                  in={expanded}
+                  timeout={'auto'}
+                >
+                  <Typography variant={'body1'} onClick={handleExpandClick}>
+                    {biography}
+                    <Link
+                      href={lastLink}
+                      style={{ margin: '1px' }}
+                      target="_blank"
+                      rel="nofollow"
+                    >
+                      Read more...
+                    </Link>
+                  </Typography>
+                </Collapse>
+                {expanded ? (
+                  <Button
+                    variant="contained"
+                    color="inherit"
+                    className={classes.expand}
+                    endIcon={<ExpandLessIcon />}
+                    onClick={handleExpandClick}
+                  >
+                    Read less
+                  </Button>
+                ) : (
+                  <Button
+                    variant="contained"
+                    color="inherit"
+                    className={classes.expand}
+                    endIcon={<ExpandMoreIcon />}
+                    onClick={handleExpandClick}
+                  >
+                    Read More
+                  </Button>
+                )}
+              </CardContent>
+            </div>
+          </Card>
+        </div>
+      </div>
+      <div className={classes.mbio}>
+        <Collapse collapsedHeight={'1.5em'} in={expanded} timeout={'auto'}>
+          <Typography variant={'body1'} onClick={handleExpandClick}>
+            {biography}
+            <Link href={lastLink} target="_blank" rel="nofollow">
+              Read more...
+            </Link>
+          </Typography>
+        </Collapse>
+        {expanded ? (
+          <Button
+            variant="contained"
+            color="inherit"
+            className={classes.less}
+            onClick={handleExpandClick}
+          >
+            less
+          </Button>
+        ) : (
+          <Button
+            variant="contained"
+            color="inherit"
+            className={classes.more}
+            onClick={handleExpandClick}
+          >
+            More
+          </Button>
+        )}
+      </div>
+    </>
+  )
+}
+
+const ArtistAlbum = ({ record, width }) => {
+  const [artist, setartist] = useState([])
+  const classes = useStyles()
+  const translate = useTranslate()
+  const [perPage] = useAlbumsPerPage(width)
+  try {
+    useEffect(() => {
+      subsonic
+        .getArtist(record?.id)
+        .then((resp) => resp.json['subsonic-response'])
```

Comment:
Don't use the Subsonic API here. If you want the artist albums, you can get it by calling the resource `album`, and filtering by `artistId`

Reply:
Thank you for the review; I've made the required changes